packages/js/src/indexables-page.js

```diff
@@ -16,7 +17,7 @@ domReady( () => {
 
 	render(
 		<Root context={ context }>
-			<IndexablesPage />
+			<LandingPage amount={ parseInt( yoastIndexingData.amount, 10 ) } />
```

Comment:
since it's a global, and I don't think we'll reuse the landingpage component, perhaps we do not need to pass the global as a prop here. Use it in the landing page component instead?